MagazineLayout/LayoutCore/SectionModel.swift

```diff
@@ -384,6 +384,13 @@ struct SectionModel {
       }
 
       indexInCurrentRow = consecutiveSameItemWidthModes % Int(startingItemWidthMode.widthDivisor) - 1
+
+      if indexInCurrentRow >= 0 {
+        for row in 0...indexInCurrentRow {
+          heightOfTallestItemInCurrentRow = max(heightOfTallestItemInCurrentRow, itemModels[startingIndex - row - 1].size.height)
+        }
+      }
```

Comment:
We should be able to figure out the tallest item in the current row in the for loop right above this. See https://github.com/airbnb/MagazineLayout/pull/42